src/wix/WixToolset.Core.Burn/Bundles/BurnCommon.cs

```diff
@@ -126,8 +130,7 @@ public BurnCommon(IMessaging messaging, string fileExe)
         public UInt32 ContainerCount { get; protected set; }
         public UInt32 UXAddress { get; protected set; }
         public UInt32 UXSize { get; protected set; }
-        public UInt32 AttachedContainerAddress { get; protected set; }
-        public UInt32 AttachedContainerSize { get; protected set; }
+        public List<ContainerSlot> AttachedContainers { get; protected set; }
```

Comment:
Investigate whether the code would be simplified if the UX container was in this as well.